drivers/hmis_csv_twenty_twenty_four/extensions/grda_warehouse/hud/project_extension.rb

```diff
@@ -0,0 +1,52 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HmisCsvTwentyTwentyFour::GrdaWarehouse::Hud
+  module ProjectExtension
+    extend ActiveSupport::Concern
+
+    included do
+      has_many :imported_items_2024, class_name: '::HmisCsvTwentyTwentyFour::Importer::Project', primary_key: [:ProjectID, :data_source_id], foreign_key: [:ProjectID, :data_source_id]
+      has_many :loaded_items_2024, class_name: '::HmisCsvTwentyTwentyFour::Loader::Project', primary_key: [:ProjectID, :data_source_id], foreign_key: [:ProjectID, :data_source_id]
+
+      def convert_to_aggregated!
+        existing = HmisCsvImporter::Aggregated::Enrollment.where(data_source_id: data_source_id, ProjectID: self.ProjectID).exists?
+        raise 'This can only be run once per data source!' if existing
+
+        data_source.update(
+          import_aggregators: {
+            'Enrollment' => ['HmisCsvImporter::Aggregated::CombineEnrollments'],
+          },
+        )
+        update(combine_enrollments: true)
+
+        copy_data_to_aggregates(enrollments, HmisCsvImporter::Aggregated::Enrollment)
+        copy_data_to_aggregates(exits, HmisCsvImporter::Aggregated::Exit)
+      end
+
+      private def copy_data_to_aggregates(scope, destination_class)
+        pseudo_importer_log = HmisCsvImporter::Importer::ImporterLog.new(id: 0)
+        scope.find_in_batches do |batch|
+          look_asides = []
+          batch.each do |row|
+            data = row.slice(row.class.hmis_structure(version: '2020').keys)
```

Comment:
Does this need updating?
```suggestion
            data = row.slice(row.class.hmis_structure(version: '2024').keys)
```